app/assets/js/controllers/basket.js

```diff
@@ -0,0 +1,23 @@
+function BasketController ($scope, basketService, universal_variable, debounceOptions) {
+	var vm = this;
+
+	vm.debounceOptions = debounceOptions;
+
+	vm.updateLineItem = function(line_item, index) {
+		universal_variable.basket.line_items[index].quantity = line_item.quantity;
```

Comment:
just feels a bit odd writing it as `universalVariable` after being so use to seeing it called `universal_variable` in all the docs and such